core/src/com/po/fuck/Assets.java

```diff
@@ -0,0 +1,35 @@
+package com.po.fuck;
+
+import com.badlogic.gdx.assets.AssetDescriptor;
+import com.badlogic.gdx.assets.AssetManager;
+import com.badlogic.gdx.graphics.Texture;
+
+public class Assets {
+
+    public AssetManager manager = new AssetManager();
+    public static final AssetDescriptor<Texture> glock3Texture =
+            new AssetDescriptor<>("glock3.png", Texture.class);
+    public static final AssetDescriptor<Texture> glock2Texture =
+            new AssetDescriptor<>("glock2.png", Texture.class);
+    public static final AssetDescriptor<Texture> basicEnemyTexture =
+            new AssetDescriptor<>("basicEnemy.png", Texture.class);
+    public static final AssetDescriptor<Texture> bulletTexture =
+            new AssetDescriptor<>("bullet2.png", Texture.class);
+    public static final AssetDescriptor<Texture> playerTexture =
+            new AssetDescriptor<>("FUCKerWithoutHands2.png", Texture.class);
+    public static final AssetDescriptor<Texture> laserGunTexture =
+            new AssetDescriptor<>("laser_gun2.png", Texture.class);
```

Comment:
Use path given in the JSON file.

Reply:
Do we need to have JSON file for every texture?